tutorials/multi/2_node_classification.py

```diff
@@ -0,0 +1,396 @@
+"""
+Single Machine Multi-GPU Minibatch Node Classification
+======================================================
+
+In this tutorial, you will learn how to use multiple GPUs in training a
```

Comment:
nit: better to make full use of 80 chars before change line.

Reply:
ok